Update steps in IG. (#1584)
codenvy/che-docs: modules/installation-guide/partials/proc_installing-the-che-operator.adoc

> {prod} Operator provides with all the resources for running {prod-short} such as PostgreSQL, Keycloak, Registries and the {prod-short} server, as well as configures all these services.

Reviewer: ```suggestion
{prod} Operator provides all the resources for running {prod-short}, such as PostgreSQL, {identity-provider}, image registries, and the {prod-short} server, and also configures all these services.
```

. In the *{prod}* pop-up window, click the btn:[Install] button.

. On the *Install Operator* screen, specify the following options:
+
* *Installation mode*: *A specific {orch-namespace} on the cluster*

> * *Installed Namespace*: *Pick an existing namespace* -> `{prod-short}`

Reviewer: Just to make sure... shouldn't the value be `{prod-id-short}` here?

Author: It is not, see my commend above.

> * *Installed Namespace*: *Pick an existing namespace* -> `{prod-short}`

Reviewer 2: ```suggestion
* *Installed Namespace*: *Pick an existing namespace* -> `{prod-namespace}`
```
